cpp/ql/src/semmle/code/cpp/Function.qll

```diff
@@ -139,6 +139,9 @@ class Function extends Declaration, ControlFlowNode, AccessHolder, @function {
   /** Gets a parameter of this function. */
   Parameter getAParameter() { params(unresolveElement(result), underlyingElement(this), _, _) }
 
+  /** Get an access of this function. */
```

Comment:
I think we should specifically call out `getACallToThisFunction()` as an alternative here, as it's likely to be the predicate a user is looking for over `getAnAccess()` a lot of the time (and experience tells us that users are often attracted to predicates with simple names such as this one).  Perhaps:
```
Gets an access of this function.

To get calls to this function, use `getACallToThisFunction` instead.
```
I've also noticed that `getACallToThisFunction` is missing a QLDoc comment itself.  I think this can be very simple for now:
```
Gets a call to this function.
```
(you can add this here, or I'll make a separate PR to address it)

Reply:
Fantastic, I've added your recommendation regarding the getACallToThisFunction thing.

Reply:
I'm not sure what is a QLDoc comment so not sure what to do here :D

Reply:
Oh gotcha, I just read it the way it was supposed to be read and noticed. Let me fix it.